appengine/flexible/cloudsql_postgresql/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Flask==0.12.2
+Flask==1.0.2
```

Comment:
That's a major version update. This should be okay, but could you verify associated documentation?